lib/Config/Initializer.php

```diff
@@ -282,8 +282,7 @@ function setupPages() {
   }
 
   function setupHooks() {
-    $hooks = new Hooks();
-    $hooks->init();
+    $this->container->get(\MailPoet\Config\Hooks::class)->init();
```

Comment:
Is this dependency on a container temporary so we don't have to refactor the whole `Initializer`?

Reply:
Yes, it is temporary. I'm doing this refactor step by step. In final state I would like move creating of container to `mailpoet_initializer.php` and create Initializer using container. Maybe the next hacking day.